lib/utils/logger/index.js

```diff
@@ -1,3 +1,19 @@
+const chalk = require('chalk');
+
+var Logger = function(){};
```

Comment:
Let's turn this into a getLogger function and pass a `transport` as the first argument. 
Which returns the object literal which I was talking about down below.

That way you can provide a mocked version of `console` in your tests. And naturally this would have more options for the future as well.

Reply:
No idea what you're talking about. This is all Chinese at the moment